libs/providers/flagd/src/e2e/step-definitions/flagd-json-evaluator.spec.ts

```diff
@@ -24,7 +24,8 @@ const evaluateStringFlagWithContext: StepsDefinitionCallbackFunction = ({ given,
     flagKey = key;
     defaultValue = defaultVal;
   });
-  and(/^a context containing a key "(.*)", with value "(.*)"$/, (key: string, value: string) => {
+  // the below has to match quotes strings ("str") and numbers (3) to test an error input
+  and(/^a context containing a key "(.*)", with value "?([^"]*)"?$/, (key: string, value: string) => {
```

Comment:
This matcher probably should have never worked when we first added `3` as an error state, it looks like it was a bug eventually fixed in `jest-gherkin`. This regex matches `"abc"` and `3`, which fixes the e2e test.